src/Compilers/CSharp/Test/Emit/PDB/PDBLambdaTests.cs

```diff
@@ -1330,5 +1330,227 @@ static void F()
 </symbols>
 ");
         }
+
+        [WorkItem(749138, "DevDiv")]
+        [Fact]
+        public void SequencePoints_Delegate1()
+        {
+            var source = @"
+using System;
+delegate void D();
+class C
+{
+    public static void Main()
+    {
+        bool b = true;
+        if (b)
+        {
+            Action a = null;
+            a = (Action)delegate { a = null; };
+            a();
+        }
+    }
+}
+";
+
+            var c = CreateCompilationWithMscorlibAndSystemCore(source, options: TestOptions.DebugDll);
+            c.VerifyPdb(@"
```

Comment:
I think it would be better to use VerifyIL(..., sequencePoints: "...") which will overlay sequence points over IL. From that it would be obvious where the sequence points are placed in relation to the display class construction.